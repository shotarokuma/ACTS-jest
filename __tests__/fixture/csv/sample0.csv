Column1	Column2	Column3	Column4	Column5	Column6	Column7	Column8	Column9	Column10
38	true	false	4	true	2	28	false	59	47	true	44	true	2	true	true	8	true	false	67