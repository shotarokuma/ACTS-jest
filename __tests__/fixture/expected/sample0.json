[
{"0":"38","1":"true","2":"false","3":"4","4":"true","5":"2","6":"28","7":"false","8":"59","9":"47","10":"true","11":"44","12":"true","13":"2","14":"true","15":"true","16":"8","17":"true","18":"false","19":"67"}
]
